modules/camera/hooks/useScanLicencePlate.ts

```diff
@@ -74,7 +75,7 @@ export const useScanLicencePlate = () => {
    * Finds the biggest block of text in the frame and checks whether it meets the criteria for ECV
    */
   const scanLicencePlate = useCallback((frameObject: TextData) => {
-    if (!frameObject.blocks) return ''
+    if (!frameObject?.blocks) return ''
```

Comment:
Hmm?

Reply:
No clue why it came as undefined... the library has the type that says frame object will be there always... but I will make the type not required in the function and write the comment